R/system.R

```diff
@@ -1176,7 +1176,67 @@ getDBConnection <- function(type, host = NULL, port = "", databaseName = "", use
         }
         conn <- eval(parse(text=connstr))
       } else if (host != "") {
-        # TODO: Implement direct connect
+        connectionString <- stringr::str_c(
+          "Driver=", driver, ";Server=", host, ";Port=", port, ";Database=", databaseName,
+          ";UID=", username, ";PWD=", password
+        );
+        if (additionalParams != "") {
+          # The "additionalParams" is passed as 'a=1,b=2,c=3'. Replace"," with ";" so that connection string because a=1;b=2;c=3
```

Comment:
Please correct the grammatical error in the comment.

Reply:
Fixed grammar error.